src/app/components/SVG/SVG.module.css

```diff
@@ -0,0 +1,15 @@
+.svg {
+  width: 2rem;
+  height: 2rem;
+  fill: #2b2b2d;
+}
+
+.svg:hover,
+.selected,
+.selected:hover {
+  fill: url(#occtyGradient);
```

Comment:
Thats fancy.. :-) Haven't seen it before.

Reply:
😉